src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -725,7 +750,7 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
   // 3% in Tau ID SF with different anti-l fake WP
   cb.cp()
       .channel({"mt", "tt"})
-      .process(JoinStr({signals, {"EMB", "ZTT", "TTT", "TTL", "VVT", "VVL"}}))
+      .process(JoinStr({signals, signals_HWW, mssm_signals, {"EMB", "ZTT", "TTT", "TTL", "VVT", "VVL"}}))
```

Comment:
Did we not apply these uncertainties on the MSSM signals so far?

Reply:
As it seems not